pkgs/development/tools/rust/crate2nix/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, makeWrapper
+
+, cargo
+, nix
+, nix-prefetch-git
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "crate2nix";
+  version = "0.8.0";
+
+  src = fetchFromGitHub
+    {
```

Comment:
```suggestion
  src = fetchFromGitHub {
```

Reply:
Thanks, but I undid this change because I use `nixpkgs-fmt`, which really doesn't like that.